docs/pages/docs/ci/github-actions.mdx

```diff
@@ -78,18 +78,13 @@ Create file called `.github/workflows/ci.yml` in your repository with the follow
               with:
                 path: |
                   node_modules/.cache/turbo
-                  # if you use pnpm, uncomment the following lines
-                  # node_modules/.pnpm
-                  # ~/.pnpm-store
-                key: ${{ runner.os }}-${{ matrix.os }}-${{ hashFiles('**/pnpm-lock.yaml') }}
-                restore-keys: |
-                  ${{ runner.os }}-${{ matrix.os }}-
-                  ${{ runner.os }}-
+                key: turbo
```

Comment:
This is not actually a viable approach as it will be created once and used indefinitely:

https://github.com/actions/cache/blob/main/src/save.ts#L36-L41